chapters/features.txt

```diff
@@ -548,14 +548,53 @@ supported properties of individual formats as normal.
     code:ImageGatherExtended capability.
   * [[features-shaderStorageImageExtendedFormats]]
     pname:shaderStorageImageExtendedFormats specifies whether all the
-    extended storage image formats are available in shader code.
-    If this feature is enabled then the
-    ename:VK_FORMAT_FEATURE_STORAGE_IMAGE_BIT feature must: be supported in
-    pname:optimalTilingFeatures for all of the extended formats.
-    To query for additional properties, or if the feature is not enabled,
-    flink:vkGetPhysicalDeviceFormatProperties and
-    flink:vkGetPhysicalDeviceImageFormatProperties can: be used to check for
-    supported properties of individual formats as normal.
+    "`storage image extended formats`" below are supported; if this feature
+    is supported, then the ename:VK_FORMAT_FEATURE_STORAGE_IMAGE_BIT must:
+    be supported in pname:optimalTilingFeatures for the following formats:
++
+  ** ename:VK_FORMAT_R16G16_SFLOAT
+  ** ename:VK_FORMAT_B10G11R11_UFLOAT_PACK32
+  ** ename:VK_FORMAT_R16_SFLOAT
+  ** ename:VK_FORMAT_R16G16B16A16_UNORM
+  ** ename:VK_FORMAT_A2B10G10R10_UNORM_PACK32
+  ** ename:VK_FORMAT_R16G16_UNORM
+  ** ename:VK_FORMAT_R8G8_UNORM
+  ** ename:VK_FORMAT_R16_UNORM
+  ** ename:VK_FORMAT_R8_UNORM
+  ** ename:VK_FORMAT_R16G16B16A16_SNORM
+  ** ename:VK_FORMAT_R16G16_SNORM
+  ** ename:VK_FORMAT_R8G8_SNORM
+  ** ename:VK_FORMAT_R16_SNORM
+  ** ename:VK_FORMAT_R8_SNORM
+  ** ename:VK_FORMAT_R16G16_SINT
+  ** ename:VK_FORMAT_R8G8_SINT
+  ** ename:VK_FORMAT_R16_SINT
+  ** ename:VK_FORMAT_R8_SINT
+  ** ename:VK_FORMAT_A2B10G10R10_UINT_PACK32
+  ** ename:VK_FORMAT_R16G16_UINT
+  ** ename:VK_FORMAT_R8G8_UINT
+  ** ename:VK_FORMAT_R16_UINT
+  ** ename:VK_FORMAT_R8_UINT
++
+[NOTE]
+.Note
+====
+pname:shaderStorageImageExtendedFormats feature only adds guarantee of
+format support, which is specified for the whole physical device.
+Therefore enabling or disabling the feature via flink:vkCreateDevice has not
```

Comment:
Maybe "has no practical effect" instead of "not"?  Other than that, I really like it.